libs/python/pythontest/pythontest.go

```diff
@@ -0,0 +1,32 @@
+package pythontest
+
+import (
+	"context"
+	"path/filepath"
+	"strings"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/python"
+	"github.com/stretchr/testify/require"
+)
+
+func RequirePythonVENV(t testutil.TestingT, ctx context.Context, pythonVersion string, checkVersion bool) string {
+	tmpDir := t.TempDir()
+	testutil.Chdir(t, tmpDir)
+
+	venvName := testutil.RandomName("test-venv-")
+	testutil.RunCommand(t, "uv", "venv", venvName, "--python", pythonVersion, "--seed")
```

Comment:
Since this depends on `uv`, could you add a `testutil.RequireUV` function to check that it is available?

Similar to `testutil.RequireJDK`.

Then `make test` won't fail if it is not installed.